schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
```

Comment:
- [ OPTIONAL] I would like to kindly suggest considering an improvement for the ID column in your database tables. It would be highly beneficial to make the ID column "not null" and auto-incremented, ensuring each record is assigned a unique ID value automatically. :win: something like this :+1:  `id int NOT NULL IDENTITY(1,1)`

Reply:
- [OPTIONAL ] I recommend marking the ID column as the primary key. By implementing these suggestions, you can enhance the efficiency and organization of your database design. :pray:  `PRIMARY KEY (id)`

Reply:
Thank you for your valuable advice.